app/data/models/payment-request.js

```diff
@@ -29,6 +29,12 @@ module.exports = (sequelize, DataTypes) => {
         return convertValueToStringFormat(this.value)
       }
     },
+    daysWaiting: {
+      type: DataTypes.VIRTUAL,
+      get () {
+        return (Math.round((Date.now() - this.received) / (1000 * 60 * 60 * 24)))
```

Comment:
'nit:
I was thinking it might be better to put these values `(1000 * 60 * 60 * 24)` into named variables to aid understanding. LMKYT.

Reply:
I don't disagree but that's not a change as part of this PR.  I've branched this from sfi-1889 branch so until that's merged into main it will show both changes in this PR.